<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Wallet</class>
 <widget class="QDialog" name="Wallet">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>757</width>
    <height>418</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Wallet</string>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>110</y>
     <width>751</width>
     <height>291</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>0</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">QTabBar::tab {
    min-width: 150px; /* Adjust the minimum width as needed */
}
</string>
   </property>
   <property name="tabPosition">
    <enum>QTabWidget::North</enum>
   </property>
   <property name="tabShape">
    <enum>QTabWidget::Rounded</enum>
   </property>
   <property name="currentIndex">
    <number>3</number>
   </property>
   <property name="iconSize">
    <size>
     <width>25</width>
     <height>25</height>
    </size>
   </property>
   <property name="elideMode">
    <enum>Qt::ElideNone</enum>
   </property>
   <property name="usesScrollButtons">
    <bool>true</bool>
   </property>
   <property name="documentMode">
    <bool>false</bool>
   </property>
   <property name="tabsClosable">
    <bool>false</bool>
   </property>
   <widget class="QWidget" name="BalanceTab">
    <attribute name="icon">
     <iconset theme="accessories-text-editor"/>
    </attribute>
    <attribute name="title">
     <string>Balance </string>
    </attribute>
    <widget class="QLCDNumber" name="BalanceShow">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>40</y>
       <width>591</width>
       <height>131</height>
      </rect>
     </property>
     <property name="smallDecimalPoint">
      <bool>false</bool>
     </property>
     <property name="digitCount">
      <number>10</number>
     </property>
     <property name="segmentStyle">
      <enum>QLCDNumber::Filled</enum>
     </property>
     <property name="value" stdset="0">
      <double>353.000000000000000</double>
     </property>
     <property name="intValue" stdset="0">
      <number>353</number>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>640</x>
       <y>40</y>
       <width>71</width>
       <height>121</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>100</pointsize>
      </font>
     </property>
     <property name="text">
      <string>$</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="Withdraw">
    <attribute name="icon">
     <iconset theme="go-down"/>
    </attribute>
    <attribute name="title">
     <string>Withdraw</string>
    </attribute>
    <widget class="QLineEdit" name="MoneyToWithdraw">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>50</y>
       <width>481</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>20</pointsize>
      </font>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="placeholderText">
      <string>Enter Amount to withdraw</string>
     </property>
    </widget>
    <widget class="QPushButton" name="WithdrawButton">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>140</y>
       <width>181</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Withdraw</string>
     </property>
    </widget>
    <widget class="QLabel" name="WithdrawLabel">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>189</y>
       <width>401</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="Deposite">
    <attribute name="icon">
     <iconset theme="go-up"/>
    </attribute>
    <attribute name="title">
     <string>Deposit</string>
    </attribute>
    <widget class="QPushButton" name="DepositButton">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>140</y>
       <width>181</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Deposit</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="MoneyToDepositeBox">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>50</y>
       <width>401</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>20</pointsize>
      </font>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="placeholderText">
      <string>Enter Money to Deposit</string>
     </property>
    </widget>
    <widget class="QLabel" name="DepositLabel">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>186</y>
       <width>381</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="Transfer">
    <attribute name="icon">
     <iconset theme="go-next"/>
    </attribute>
    <attribute name="title">
     <string>Transfer</string>
    </attribute>
    <widget class="QPushButton" name="TransferButton">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>160</y>
       <width>181</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Transfer</string>
     </property>
    </widget>
    <widget class="QLabel" name="TransferStatusLabel">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>200</y>
       <width>381</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>20</y>
       <width>391</width>
       <height>131</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QLineEdit" name="ReceiverUsernameBox">
        <property name="font">
         <font>
          <pointsize>18</pointsize>
         </font>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
        <property name="placeholderText">
         <string>Enter Username</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="MoneyToTransfer">
        <property name="font">
         <font>
          <pointsize>20</pointsize>
         </font>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
        <property name="placeholderText">
         <string>Enter the Amount of Money</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QWidget" name="Logs">
    <attribute name="icon">
     <iconset theme="text-x-script"/>
    </attribute>
    <attribute name="title">
     <string>Account Logs</string>
    </attribute>
    <widget class="QListWidget" name="LogList">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>721</width>
       <height>171</height>
      </rect>
     </property>
     <property name="verticalScrollBarPolicy">
      <enum>Qt::ScrollBarAlwaysOn</enum>
     </property>
     <property name="isWrapping" stdset="0">
      <bool>false</bool>
     </property>
     <property name="wordWrap">
      <bool>false</bool>
     </property>
     <property name="selectionRectVisible">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="UndoButton">
     <property name="geometry">
      <rect>
       <x>440</x>
       <y>220</y>
       <width>131</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Undo</string>
     </property>
    </widget>
    <widget class="QPushButton" name="RedoButton">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>220</y>
       <width>131</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Redo</string>
     </property>
    </widget>
    <widget class="QLabel" name="LogLabel">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>190</y>
       <width>671</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>0</y>
     <width>181</width>
     <height>91</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Welcome to Smart Wallet</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="GreetingLabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>0</y>
     <width>331</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Hello, Mr. Abdallah Hossam</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="RefreshButton">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>80</y>
     <width>89</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Refresh</string>
   </property>
  </widget>
  <widget class="QLabel" name="RefreshLabel">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>80</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
